pynetdicom/dimse_messages.py

```diff
@@ -161,6 +163,11 @@
     'N_DELETE' : N_DELETE,
 }
 
+_REPEATING_TAGS = [
+    Tag(tag_for_keyword('OffendingElement')),
```

Comment:
Just use the int form of the UID, its much faster than the lookup methods in pydicom

Reply:
Also, can you change the name of the variable? Repeating tags have a specific meaning. `_MULTIVALUE_TAGS` or something.

Reply:
Wow. Never noticed that `Tag(<keyword>)` was added to the api. Good to know :)